pytorch_lightning/trainer/properties.py

```diff
@@ -118,16 +118,13 @@ def is_global_zero(self) -> bool:
 
     @property
     def slurm_job_id(self) -> Optional[int]:
-        try:
-            job_id = os.environ['SLURM_JOB_ID']
+        job_id = os.environ.get('SLURM_JOB_ID')
+        if job_id:
```

Comment:
This will fail if `SLURM_JOB_ID` is not a number string

Reply:
well, do we expect that this can happen?

Reply:
Probably not if everything works as expected but i'd say better safe than sorry

Reply:
fixed with adding ValueError